docs/reference-guide/triggers.md

```diff
@@ -4,20 +4,17 @@ title: Triggers
 sidebar_label: Triggers
 ---
 
-This article is a part of a series intended to showcase Memgraph's features and
-bring the user up to speed on developing with Memgraph.
+[![Related -How-to](https://img.shields.io/static/v1?label=Related&message=How-to&color=blue&style=for-the-badge)](/how-to-guides/how-to-setup-triggers.md)
```

Comment:
Unfortunately, we can't have these labels as a first thing on the page because when we paste links, the short description turns out to be markdown code, so we try to first have a little intro, then add this label... so just switch up the text and the label